server/app.js

```diff
@@ -16,9 +16,15 @@ app.use(bodyParser.urlencoded({ extended: true }));
 
 const port = process.env.PORT || 6000;
 
+app.get('/', (req, res) => res.status(200).json({
+  status: 200,
+  message: 'Welcome to Politico',
+}));
+
 app.use('/api/v1/', routes);
 
 app.all('*', (req, res) => res.status(404).json({
+  status: 404,
   message: 'Wrong endpoint. Such endpoint does not exist',
```

Comment:
Please check the project document for the correct response message format for errors.